docs/Tutorials/Create-Permissioned-Network.md

```diff
@@ -0,0 +1,349 @@
+        "privateKey": "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63",
+        "comment": "private key and this comment are ignored.  In a real chain, the private key should NOT be stored",
+        "balance": "0xad78ebc5ac6200000"
+      },
+      "627306090abaB3A6e1400e9345bC60c78a8BEf57": {
+        "privateKey": "c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3",
+        "comment": "private key and this comment are ignored.  In a real chain, the private key should NOT be stored",
+        "balance": "90000000000000000000000"
+      },
+      "f17f52151EbEF6C7334FAD080c5704D77216b732": {
+        "privateKey": "ae6ae8e5ccbfb04590405997ee2d52d2b330726137b875053c36d94e974d162f",
+        "comment": "private key and this comment are ignored.  In a real chain, the private key should NOT be stored",
+        "balance": "90000000000000000000000"
+      }
+   },
+  "number":"0x0",
+  "gasUsed":"0x0",
+  "parentHash":"0x0000000000000000000000000000000000000000000000000000000000000000"
+}
+```
+
+In `extraData`, replace `<Node 1 Address>` with the [address for Node-1](#3-get-address-for-node-1) excluding the 0x prefix. 
+
+!!! example
+    
+    ```json
+    {
+      ...
+    "extraData":"0x0000000000000000000000000000000000000000000000000000000000000000b9b81ee349c3807e46bc71aa2632203c5b4620340000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+      ...
+    }
+    ```
+
+!!! warning 
+    Do not use the accounts in the genesis file on MainNet or any public network except for testing.
+        
+    The private keys are displayed which means the accounts are not secure.  
+
+### 5. Create Permissions Configuration File 
+
+The permissions configuration file defines the nodes and accounts whitelists. 
+
+Copy the following permissions configuration to a file called `permissions_config.toml` and save a copy in the `Node-1-data-path`, 
+`Node-2-data-path`, and `Node-3-data-path` directories:
+
+!!! example "permissions_config.toml"
+    ```toml 
+    accounts-whitelist=["0xfe3b557e8fb62b89f4916b721be55ceb828dbd73", "0x627306090abaB3A6e1400e9345bC60c78a8BEf57"]
+     
+    nodes-whitelist=["enode://<publicKeyNode1 ex 0x>@127.0.0.1:30303","enode://<publicKeyNode2 ex 0x>@127.0.0.1:30304","enode://<publicKeyNode3 ex 0x>@127.0.0.1:30305"]
+    ```
+
+Replace the public key as indicated for each enode URL with the [node public keys](#2-get-node-public-keys). 
+
+The permissions configuration file includes: 
+
+* First two accounts from the genesis file.
+* Enode URLs for each node. 
+
+!!! note
+    Permissions are specified at the node level. The [`permissions_config.toml`](../Permissions/Permissioning.md#permissions-configuration-file) 
+    file must be saved in the data directory for each node. 
+    
+    On-chain permissioning is under development. On-chain permissioning will use one on-chain 
+    nodes whitelist. 
```

Comment:
one nodes whitelist and one accounts whitelist

Reply:
Done